src/db/models/Commit.ts

```diff
@@ -34,3 +35,15 @@ export async function getCommitsAfterDate(afterDate: Date): Promise<Commit[]> {
     .select("*")
     .where<Commit[]>("commit.committed_at", ">", afterDate.toISOString());
 }
+
+export async function getTopCommitsAfterDate(
+  afterDate: Date,
+  limitCount: number
+): Promise<Commit[]> {
+  return db
+    .from<Commit>(tableName)
+    .select("*")
+    .where("commit.committed_at", ">", afterDate.toISOString())
+    .orderBy("commit.score", "desc")
```

Comment:
Include a tie-breaker field in the ordering, otherwise the order of returned commit (with same score) is undefined.